ДЗ-2

Dmitrii64 opened the pull request:
Сделал main и catalog

Dmitrii64 committed:
ДЗ-2

Сделал main и catalog

tahavn reviewed (changes requested):
Проект не запускает.
Выполни требования 2-го ДЗ, а после этого уже можешь по своему усмотрению создавать дополнительные компоненты.
http://joxi.ru/xAe6Y6VHg7eEPr что за странные параметры, уже вроде обсуждали что там в структуре можно обойтись без br и соответственно использовать один параметр для текста.
проект не запускается из-за отсутствующего файла index.js в папке catalog

Dmitrii64 committed:
не работают страницы

Dmitrii64 committed:
Исправил

Требования ДЗ-2 выполнил. Единственное, что не очень красиво получилось - это подзаголовок на странице toys и лишняя кнопка на странице contact us (но это не входило в дз, это доп. компоненты). Я просто пока не очень понимаю, как поступать когда целый кусок кода повторяется (форма или подзаголовок), но на одной из страниц есть доп. элементы внутри. Это дз можно закрыть, чтобы я двигался дальше, а эти доп. компоненты я потом приведу в порядок?

tahavn reviewed (commented):
дз-2 (90%)
